checker/bin/wpi-annotation-paths.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+# Given the name of a directory containing the results produced by an
+# invocation of wpi-many.sh and a filename, output the paths to the .ajava
+# files produced by whole-program inference to a .txt file with the given name.
```

Comment:
Why output to a `.txt` file? In the standard unix scripting style, I'd expect a script like this to output to stdout; users can easily pipe the result to a file (the same way that you do!) or do something else with it.

Reply:
Also, this comment claims that the output will go to a .txt file, but that isn't enforced at all: the user can supply any kind of file that they'd like.